packages/familie-http/src/axios.ts

```diff
@@ -0,0 +1,90 @@
+import { captureException, configureScope, withScope } from '@sentry/core';
+import axios, { AxiosError, AxiosResponse } from 'axios';
+
+import { Ressurs, RessursStatus, ApiRessurs, ISaksbehandler } from '@navikt/familie-typer';
+
+axios.defaults.baseURL = window.location.origin;
+export const preferredAxios = axios;
+
+export interface HåndterApiRessurs<T> {
+    ressurs?: ApiRessurs<T>;
+    innloggetSaksbehandler?: ISaksbehandler;
+    error?: AxiosError;
+}
+
+export const håndterApiRessurs = <T>(håndterApiRessurs: HåndterApiRessurs<T>): Ressurs<T> => {
+    const { ressurs, innloggetSaksbehandler, error } = håndterApiRessurs;
+    let typetRessurs: Ressurs<T>;
+
+    if (!ressurs) {
+        return {
+            frontendFeilmelding: 'En feil har oppstått!',
+            status: RessursStatus.FEILET,
+        };
+    }
+
+    switch (ressurs.status) {
+        case RessursStatus.SUKSESS:
+            typetRessurs = {
```

Comment:
Istedenfor 

```
let typetRessurs = ...
case...
break;
```

kan vi kanskje returnere direkte for hvert case?

Reply:
Dessverre forstår ikke ts at returtypen stemmer og det er derfor lagt til en eksplisitt type her :)

Reply:
Of course - tenkte ikke på det 🙈